### PR TITLE
Add absolute number to number range

### PR DESCRIPTION
**Pull request by amansrivastava17**
Number range detector will now check for absolute values in the input text and return it in abs_value.
Why? When we ask for the user's budget, he might say, "1k-2k rupees" or simply "2k rupees"
Ex: input:
detctor.detect_entity(text='10 rupees to 30, more than 75 rupees 85 rupees')
output:
([{'abs_value': None, 'max_value': '30', 'min_value': '10', 'unit': u'rupees'},
{'abs_value': None, 'max_value': None, 'min_value': '75', 'unit': u'rupees'},
{'abs_value': '85', 'max_value': None, 'min_value': None, 'unit': 'currency'}],
['10 rupees to 30', 'more than 75 rupees', '85 rupees'])

### CODE REVIEW

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by amansrivastava17** [comment]
retest this please

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by SinghDivneet** [comment]
RETEST this please

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 1]
```diff
@@ -179,8 +179,33 @@ def detect_number_range(self, text):
...
+        abs_number_pattern = re.compile(ur'({number}\d+)'.format(number=numeral_constant.NUMBER_REPLACE_TEXT),
+                                        re.UNICODE)
+        abs_number_matches = abs_number_pattern.findall(self.processed_text)
+        for match in abs_number_matches:
+            if self.unit_type:
+                if self.number_detected_map[match].entity_value[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT] \
+                        is not None:
+                    number_abs_list.append({numeral_constant.NUMBER_RANGE_MAX_VALUE: None,
+                                            numeral_constant.NUMBER_RANGE_MIN_VALUE: None,
+                                            numeral_constant.NUMBER_RANGE_VALUE_UNIT: self.unit_type,
```
Comment (amansrivastava17) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: correct unit type, it should be equal to `self.number_detected_map[match].entity_value[numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]`

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 2]
```diff
@@ -181,6 +182,24 @@ def detect_number_range(self, text):
...
+        original_list = original_list or []
+        abs_number_pattern = re.compile(ur'({number}\d+)'.format(number=numeral_constant.NUMBER_REPLACE_TEXT),
+                                        re.UNICODE)
+        abs_number_matches = abs_number_pattern.findall(self.processed_text)
+        for match in abs_number_matches:
+            entity_unit = self.number_detected_map[match].entity_value[
+                numeral_constant.NUMBER_DETECTION_RETURN_DICT_UNIT]
+            if (self.unit_type and entity_unit == self.unit_type) or not self.unit_type:
+                number_list.append({numeral_constant.NUMBER_RANGE_MAX_VALUE: None,
+                                        numeral_constant.NUMBER_RANGE_MIN_VALUE: None,
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: E127 continuation line over-indented for visual indent

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 3]
```diff
@@ -181,6 +182,24 @@ def detect_number_range(self, text):
...
+                                        numeral_constant.NUMBER_RANGE_VALUE_UNIT: entity_unit,
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: E127 continuation line over-indented for visual indent

File: ner_v2/detectors/numeral/number_range/standard_number_range_detector.py [hunk 4]
```diff
@@ -181,6 +182,24 @@ def detect_number_range(self, text):
...
+                                        numeral_constant.NUMBER_RANGE_ABS_VALUE: self.number_detected_map[
```
Comment (srajdev) [inline ner_v2/detectors/numeral/number_range/standard_number_range_detector.py]: E127 continuation line over-indented for visual indent

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Pull request merged**